e2e/tests/apis.bats

```diff
@@ -48,7 +48,7 @@ load variables
 }
 
 @test "Test APIs with HTTPS ONLY" {
-    run dispatch create api api-test-https-only func-nodejs6 -m POST --https-only -p /https-only -a public
+    run dispatch create api api-test-https-only func-nodejs6 -m POST --https-only -p /https-only ---authuth public
```

Comment:
`---authuth` looks like it's misspelled

Reply:
looks like an effect of replacing `-a` with `--auth`

Reply:
thanks Ivan, will fix this a replace mistake